### PR TITLE
Loading Icon on Create Class

### PR DESCRIPTION
**Pull request by dividedsky**
Displayed when sendgrid/db ops are running

### CODE REVIEW

File: client/src/components/classesPage/ClassCreateView.js [hunk 1]
```diff
@@ -22,6 +23,24 @@ const styles = theme => ({
...
+    backgroundColor: 'white',
+    filter: 'blur(2px)',
+    height: '100vh',
+    width: '100vh'
+  },
+  loading: {
+    position: 'absolute',
+    top: '40%',
+    color: 'blue',
+    zIndex: '1'
```
Comment (cotikor) [inline client/src/components/classesPage/ClassCreateView.js]: Nice working getting this up

File: client/src/components/classesPage/ClassCreateView.js [hunk 2]
```diff
@@ -99,96 +119,106 @@ function ClassCreateView(props) {
...
-          const studentRes = await ax.post('/students', recipient);
-          console.log(studentRes);
-
-          // add student and class to students_classes
-          ax.post(`/classes/${list}/students`, {
-            student_id: recipient_id
+    try {
+      setIsLoading(true);
+      let firstPass = true; // flag to ensure we don't submit the same list to sendgrid
+      let list; // to store list id
```
Comment (cotikor) [inline client/src/components/classesPage/ClassCreateView.js]: Good logic here!

File: client/src/components/classesPage/ClassCreateView.js [hunk 3]
```diff
@@ -363,33 +393,46 @@ function ClassCreateView(props) {
...
-          campaignData={campaignData}
-          setCampaignData={setCampaignData}
-          stage={stage}
-          setStage={setStage}
-          sendAllToSendgrid={sgDb}
-          // setTimeData={setTimeData}
-          timeTriData={timeTriData}
-          setTimeTriData={setTimeTriData}
+    <>
+      {isLoading && (
```
Comment (cotikor) [inline client/src/components/classesPage/ClassCreateView.js]: I'll have to start using this &&, it's really clean

Comment (dividedsky) [reply client/src/components/classesPage/ClassCreateView.js]: Yes! It's really helpful in a lot of cases.

**Pull request merged**